development/towncrier_template.j2

```diff
@@ -25,4 +25,4 @@
 No significant changes.
 
 {% endif %}
-{% endfor %}
+{% endfor %}
```

Comment:
This should probably be replicated in `develop` to stop making markdownlint unhappy.

Reply:
I see it now. Good call